### ISSUE TITLE
[Functional - Bot Framework Web Chat - Card Richmessage]: 'Comment' control is not functional present under 'Card Richmessage'

### ISSUE THREAD

**Issue by manojs5** [microsoft/BotFramework-WebChat#5356]
### Is it an issue related to Adaptive Cards?

No.

### User Experience:
User will get confused with the functionality of the control and cannot access the control efficiently if the control is not behaving according to the functionality of the control.
### Note:
User credentials should NOT be included in the bug.

### What browsers and screen readers do this issue affect?

Windows: Edge with Windows Narrator

### Are there any code-based customization done to Web Chat?

No, I am using Web Chat without any customizations except "styleOptions".

### What version of Web Chat are you using?

Latest production

### Which area does this issue affect?

Suggested action

### What is the public URL for the website?

https://compulim.github.io/webchat-loader/

### How to reproduce the issue?

1. Open URL: https://compulim.github.io/webchat-loader/ and sign in using valid credentials.
2. Navigate to web chat page and select 4.18.1-main.20240927.ea7a875 (30/9/2024) version and then select mockBot link under present label and then select open web chat in new window link at bottom.
3. Navigate to white label experience chat page and type Card Richmessage word in type your message text box.
4. Navigate to the search results.
5. Verify the comment is functional or not.

### What do you expect?

'Comment' control should be functional present under 'Card Richmessage' 
Ex:
using keyboard and mouse comment action should be performed.

### What actually happened?

'Comment' control is not functional present under 'Card Richmessage' 
Observation:
Using keyboard and mouse No action performed after invoking the 'Comment' control 

### Do you have any screenshots or recordings to repro the issue?

![Image](https://github.com/user-attachments/assets/9b65b386-82b3-4e2a-bf26-f020834ff9ed)
https://github.com/user-attachments/assets/f1e194a7-5bcd-42ad-90f7-da4e59a76aa5

### WAI-ARIA reference

https://www.w3.org/WAI/ARIA/apg/patterns/button/

**Comment by manojs5** [comment]
#A11ySeV1;#Accessibility;;#Benchmark;#BotFrameworkWebChat-Oct24;#HCL;#Functional:#Win11-Edge(Chromium);
